apps/modernization-api/src/main/java/gov/cdc/nbs/event/search/InvestigationFilter.java

```diff
@@ -94,8 +94,8 @@ public enum NotificationStatus {
         UNASSIGNED(InvestigationFilter.UNASSIGNED),
         APPROVED("APPROVED"),
         COMPLETED("COMPLETED"),
-        MESSAGE_FAILED("MESSAGE_FAILED"),
-        PENDING_APPROVAL("PENDING_APPROVAL"),
+        MSG_FAIL("MSG_FAIL"),
```

Comment:
issue: changing the enum name is going to result in the name changing in the dropdown on the frontend as it uses the enum to populate the `options`.  

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/e174bebe-9437-45ed-b581-9622136aad64)

You might be able to just change the values to what the database is expecting and then change the filter to search on the `.value()` instead of `.name()`.

Reply:
technically those are the status's in the db though.  what does legacy look like?  I don's see a "Pending Approval" string anywhere in legacy code.

Reply:
ah saw the vid... they translated the statuses